app/editor/src/features/admin/av-overviews/OverviewSubscribers.tsx

```diff
@@ -0,0 +1,64 @@
+import { useFormikContext } from 'formik';
+import React from 'react';
+import { useUsers } from 'store/hooks/admin';
+import { FlexboxTable, ITableInternal, ITablePage, ITableSort, IUserModel } from 'tno-core';
+
+import { subscriberColumns } from './constants';
+import { IAVOverviewTemplateForm } from './interfaces';
+import { ListFilter } from './ListFilter';
+
+/**
+ * The page used to view and edit reports.
+ * @returns Component.
+ */
+export const OverviewSubscribers: React.FC = () => {
+  const { values, setFieldValue } = useFormikContext<IAVOverviewTemplateForm>();
+  const [{ users }, { findUsers }] = useUsers();
+
+  React.useEffect(() => {
+    async function getUsers() {
+      await findUsers({ page: 1, quantity: users.quantity });
+    }
+    getUsers();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const handlePageChange = React.useCallback(
+    async (page: ITablePage, table: ITableInternal<IUserModel>) => {
+      await findUsers({ page: page.pageIndex + 1, quantity: page.pageSize });
+    },
+    [findUsers],
+  );
+
+  const handleSortChange = React.useCallback(
+    async (sort: ITableSort<IUserModel>[], table: ITableInternal<IUserModel>) => {
+      const sorts = sort
+        .filter((s) => s.isSorted)
+        .map((s) => `${s.id}${s.isSortedDesc ? ' desc' : ''}`);
+      await findUsers({ page: 1, quantity: users.quantity, sort: sorts });
```

Comment:
Add `try+catch` wrap or the console will throw errors `try { ... } catch { // errors are already handled }`

Reply:
Updated, thanks